### DISCUSSION TITLE
NSFW Art shows as cover image for an album.

### DISCUSSION THREAD

**Discussion by tomny-dev** [chevereto/chevereto#49] [Q&A]
An album with NSFW art has the NSFW cover image showing even when NSFW is disabled. I'm wondering if this could be blurred or if albums can be marked as NSFW.

**Comment by rodber** [comment]
> An album with NSFW art has the NSFW cover image showing even when NSFW is disabled.

Exactly where? I checked on listings and works fine. Provide an screenshot to determine where is the issue.
